.github/workflows/build_deploy_docs_develop.yml

```diff
@@ -51,9 +51,23 @@ jobs:
         run:
           poetry run mkdocs build -f ./docs/mkdocs.yml -d ../site
 
-      - name: Deploy
-        uses: JamesIves/github-pages-deploy-action@v4
+      - name: Setup git config
+        run: |
+          git config user.name github-actions[bot]
+          git config user.email 41898282+github-actions[bot]@users.noreply.github.com
+
+      - name: Rename folder and commit
+        run: |
+            echo "Renaming site folder and committing"
+            git fetch
```

Comment:
You can specify in the checkout action that it should fetch all. I think with `depth: 0`, to avoid the fetch here

Reply:
Ah, I see. I'm learning...